content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -1,64 +1,78 @@
 ---
-title: Install Istio with the Istio CNI plugin
-description: Install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
+title: Install the Istio CNI component
+description: Install and use the Istio CNI component, allowing operators to deploy workloads with lower privilege.
 weight: 70
 aliases:
-    - /docs/setup/kubernetes/install/cni
     - /docs/setup/kubernetes/additional-setup/cni
+    - /docs/setup/additional-setup/cni
 keywords: [cni]
 owner: istio/wg-networking-maintainers
 test: yes
 ---
 
-Follow this guide to install, configure, and use an Istio mesh using the Istio Container Network Interface
-([CNI](https://github.com/containernetworking/cni#cni---the-container-network-interface))
-plugin.
+`istio-cni` is a Istio node agent DaemonSet that runs in your cluster. It is used by both Istio {{< gloss >}}data plane{{< /gloss >}} modes.
+
+`istio-cni` runs in your cluster with elevated privileges, and is used to configure traffic redirection
+for pods in the Istio mesh.
+
+For the {{< gloss >}}sidecar{{< /gloss >}} data plane mode, it is optional, and removes the requirement of running privileged init containers in every pod in the mesh, replacing that model with a single privileged node agent pod on each Kubernetes node.
+
+For the {{< gloss >}}ambient{{< /gloss >}} data plane mode, it is the only supported mechanism, and must be installed.
+
+This guide is focused on using the `istio-cni` component as an optional part of the sidecar data plane mode. Consult [the ambient docs](/docs/ambient/) for information on using the ambient data plane mode.
+
```

Comment:
put a heading here

```suggestion
# `istio-cni` and CNI plugins
```

(p.s. it bugs me that we use `backticks` throughout.  Do we have a precedent that would just let us call it istio-cni?  We don't do the same with istiod, do we? We do call it "Istiod", which I don't like.  I tried to always use lowercase-zed in ztunnel.)

Reply:
I think we have the headings correct here now? It looks reasonable the way we have it here anyway, LMK if you want additional heading tweaks here.

Reply:
reined a couple in